Makefile

```diff
@@ -130,11 +130,12 @@ build-tarball: install
 	# calculate md5sum and create file with its value
 	md5sum ${TGZ_NAME} > ${TGZ_NAME}.md5
 
+SWIFT_TARGET_NAME=serial-vault-builds/$(VERSION)/serial-vault.tar.gz
```

Comment:
In order to pull the tarball from swift with `ols.layer` the path needs to have a specific format, so I change it here